source/core/Events.cpp

```diff
@@ -568,7 +569,6 @@ void RegisterForStaticEvents(VirtualInstrument *vi) {
                     viName->AppendSubString(&encodedSubstr);
                     char *tagBegin = reinterpret_cast<char*>(tag->Begin()), *tagEnd = reinterpret_cast<char*>(tag->End());
                     EventControlUID controlID = Int32(strtol(tagBegin, &tagEnd, 10));
```

Comment:
> EventControlUID controlID = Int32(strtol(tagBegin, &tagEnd, 10));

1) I thought we should not parse control ID string to an integer as it can be GUID ?
2) EventControlUID should not be an integer too.
3) Why not use the term controlTag as used in ControlRefInfo (instead of UID) as we are referring to the same.  Or, change controlTag in ControlRefInfo also.

Reply:
1 and 2. I can't remove that until jsRegisterForControlEvent is refactored to use a larger (or string) GUID.  When that happens, we just change the argument typeIt's only remembered to pass for that argument though.
3. I'l get rid of EventControlUID altogether, rather than renaming it, and just use the controlTag in the refnum, when the control ID->GUID change happens.